HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/AppCoordinator.swift

```diff
@@ -330,6 +330,10 @@ extension AppCoordinator: GiniHealthDelegate {
     
     func didCreatePaymentRequest(paymentRequestID: String) {
         print("✅ Created payment request with id \(paymentRequestID)")
+        DispatchQueue.main.async {
+            (self.childCoordinators.first as! InvoicesListCoordinator).invoicesListViewController
```

Comment:
Recommendation: To avoid using forced unwrap, I suggest using this approach:

`guard let invoicesListCoordinator = self.childCoordinators.first as? InvoicesListCoordinator else {
                return
            }
            invoicesListCoordinator.invoicesListViewController.presentedViewController?.dismiss(animated: true)`